src/solver.cpp

```diff
@@ -417,6 +425,8 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     RegisterWithSolver(registry, ++id, ConvMlirIgemmFwdXdlops{}, miopenConvolutionAlgoImplicitGEMM);
     RegisterWithSolver(registry, ++id, ConvMlirIgemmBwdXdlops{}, miopenConvolutionAlgoImplicitGEMM);
     RegisterWithSolver(registry, ++id, ConvMlirIgemmWrWXdlops{}, miopenConvolutionAlgoImplicitGEMM);
+
+    Register(registry, ++id, SolverDbId(activ::ActivFwdSolver0{}));
```

Comment:
If we have single registry for all primitives, won't this affect performance?

Reply:
One would need hundreds of thousands of solvers to nothing something even on synthetic tests.